en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

```diff
@@ -237,7 +230,7 @@ Update the `choreo-connect-1.x.x/docker-compose/choreo-connect(-with-apim)/conf/
 
 ### Step 3 - Publish an API
 
-Publish an API to the Choreo Connect by either using `APIM` or `APICTL`. 
+Publish an API to Choreo Connect either via API Manager or apictl. 
```

Comment:
shall we remove this apictl based approach from docs other than for qsg @pubudu538

Reply:
And we could say if you are running on immutable gateway mode, you can simply start the gateway.

Reply:
Then the question comes how do you create the API Project without the apictl. Basically we need apictl as well. We can't skip that.